controllers/managedfusion_controller.go

```diff
@@ -432,6 +444,16 @@ func (r *ManagedFusionReconciler) reconcilePrometheus() error {
 
 		r.prometheus.Spec = desired.Spec
 		r.prometheus.Spec.ExternalLabels["clusterId"] = string(clusterVersion.Spec.ClusterID)
+
+		if r.remoteWriteConfigData != nil {
+			if r.remoteWriteConfigData.BearerToken == "" {
+				return fmt.Errorf("bearer token for remote write config data is invalid")
```

Comment:
Let's keep invalid field error similar
```suggestion
				return fmt.Errorf("invalid remote write bearer token, empty string")
```